lib/puma/dsl.rb

```diff
@@ -828,7 +828,10 @@ def wait_for_less_busy_worker(val=0.005)
     #    `set_remote_address header: "X-Real-IP"`.
     #    Only the first word (as separated by spaces or comma) is used, allowing
     #    headers such as X-Forwarded-For to be used as well.
-    # 4. **\<Any string\>** - this allows you to hardcode remote address to any value
+    # 4. **proxy_protocol: v1**- set the remote address to the value read from the
```

Comment:
Should this be

```suggestion
    # 4. **proxy_protocol: :v1**- set the remote address to the value read from the
```

?

Reply:
It gets `.to_sym` called on it, so :v1 and "v1" both work, but I can recommend the sym form if you'd prefer!